utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt

```diff
@@ -70,6 +70,8 @@ class DataProvidersTest {
   @Inject
   lateinit var asyncDataSubscriptionManager: AsyncDataSubscriptionManager
 
+  @Inject lateinit var fakeExceptionLogger: FakeExceptionLogger
```

Comment:
We are doing `Reformat code`, right? `@Inject` and other stuff in the next line.  

```
@Inject 
lateinit var fakeExceptionLogger: FakeExceptionLogger
```

Reply:
Good catch! This was another PR that was merged recently, after merging from develop I didn't reformat and so this got missed. Reformatted now